lib/Service/SnoozeService.php

```diff
@@ -108,6 +108,8 @@ public function snoozeThread(Message $selectedMessage, int $unixTimestamp): void
 			$selectedMessage->getThreadRootId()
 		);
 
+		$messages = array_unique($messages);
```

Comment:
could it make sense to adjust `\OCA\Mail\Db\ThreadMapper::findMessageIdsByThreadRoot` with a `GROUP BY` and let the database deduplicate?

it's a nano optimization because there will probably rarely be more than one duplicate

Reply:
:white_check_mark: